### PR TITLE
NAS-102484 / 12.0 / Nas 102484

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/helptext/global-helptext.ts [hunk 1]
```diff
@@ -61,5 +61,7 @@ scheduler: {
         header: T('Days'),
         tooltip: T('Days when this task will run.'),
     }
-}
+},
+
+invalidInputValueWithUnit: T('Invalid value. Missing numerical value or invalid numerical value/unit'),
```
Comment (Mrt134) [inline src/app/helptext/global-helptext.ts]:
```suggestion
invalidInputValueWithUnit: T('Invalid value. Missing numerical value or invalid numerical value/unit.'),
```

**Comment by erinix** [comment]
We should probably document how this is to be used, the user might not know to type in M/m/y/d/etc.  Could you come up with some tooltip text for this @freenas/docs-team ?

File: src/app/helptext/task-calendar/snapshot/snapshot-form.ts [hunk 2]
```diff
@@ -17,8 +17,8 @@ export default {
  Example: <i>pool1/dataset1/child1</i>. A recursive snapshot of\
  <i>pool1/dataset1</i> will include all child datasets except <i>child1</i>.'),
 
-    lifetime_value_placeholder: T('Snapshot Lifetime'),
-    lifetime_unit_tooltip: T('Define a length of time to retain the snapshot on this\
+    lifetime_placeholder: T('Snapshot Lifetime'),
+    lifetime_tooltip: T('Define a length of time to retain the snapshot on this\
  system. After the time expires, the snapshot is removed. Snapshots which have\
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/snapshot/snapshot-form.ts]: Here's a suggestion for slightly expanding this help text:

    lifetime_tooltip: T('Define a length of time to retain the snapshot \
 on this system. Does not accept Minute values. After the time expires, \
 the snapshot is removed. Snapshots which have been replicated to other \
 systems are not affected.'),

File: src/app/helptext/task-calendar/snapshot/snapshot-form.ts [hunk 3]
```diff
@@ -17,8 +17,8 @@ export default {
  Example: <i>pool1/dataset1/child1</i>. A recursive snapshot of\
  <i>pool1/dataset1</i> will include all child datasets except <i>child1</i>.'),
 
-    lifetime_value_placeholder: T('Snapshot Lifetime'),
-    lifetime_unit_tooltip: T('Define a length of time to retain the snapshot on this\
+    lifetime_placeholder: T('Snapshot Lifetime'),
+    lifetime_tooltip: T('Define a length of time to retain the snapshot on this\
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/snapshot/snapshot-form.ts]: Here's a suggestion for expanding this help text to explain what is accepted in this field:

lifetime_tooltip: T('Define a length of time to retain the snapshot \
 on this system using a numeric value and a single lowercase letter for \
 units. Examples: <i>3h</i> is three hours, <i>1m</i> is one month, and \
 <i>1y</i> is one year. Does not accept Minute values. After the time \
 expires, the snapshot is removed. Snapshots which have been replicated \
 to other systems are not affected.'),

**Pull request merged**